cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/Cargo.toml

```diff
@@ -13,15 +13,13 @@ workspace = true
 substrate-wasm-builder = { optional = true, workspace = true, default-features = true }
 
 [dependencies]
-codec = { features = [
-	"derive",
-], workspace = true }
+codec = { features = ["derive"], workspace = true }
 hex-literal = { workspace = true, default-features = true }
 log = { workspace = true }
-scale-info = { features = [
+scale-info = { features = ["derive"], workspace = true }
```

Comment:
nit: why codec vs serde formatting is inconsistent (single line vs multi)? I also noticed that in other files.

Reply:
I don't know. Even though I fix them but they will become multi line once I save the file.

Reply:
could it be your editor settings?

Reply:
Which one?

Reply:
> Which one?

Even Better TOML

Reply:
Ok, so I run this in the main repo on your branch: `env TAPLO_CONFIG=.config/taplo.toml taplo fmt cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/Cargo.toml` the formatting stays the same. I think the previous command I mentioned on this thread didn't load correctly the `taplo.toml` file and resulted in taplo applying some defaul formatting, not the one from the repo, and I am thinking your extension did the same.

I guess it requires to be configured to point to the `.config/taplo.toml` from the repo to do the expected formatting, or you already did it by now.